packages/truffle/test/scenarios/commands/ethpm.js

```diff
@@ -1,27 +1,26 @@
-var MemoryLogger = require("../MemoryLogger");
-var CommandRunner = require("../commandRunner");
-var fs = require("fs");
-var path = require("path");
-var assert = require("assert");
-var Server = require("../server");
-var Reporter = require("../reporter");
-var sandbox = require("../sandbox");
-var log = console.log;
+const MemoryLogger = require("../MemoryLogger");
+const CommandRunner = require("../commandRunner");
+const fs = require("fs");
+const path = require("path");
+const assert = require("assert");
+const Server = require("../server");
+const Reporter = require("../reporter");
+const sandbox = require("../sandbox");
+const log = console.log;
```

Comment:
The tests in this module are `skipped`. Do we need to update it if the plan is to strip out ethpm feature?